### PR TITLE
New doc: Migrate from WP Engine to Pantheon

### LINKED ISSUE #847: New Doc Proposal: Migrating WordPress sites from WPEngine
## Description
Explain the deviations from the normal process required to migrate sites from WPEngine to Pantheon

## Outline
Credit http://brianjackson.io/migrating-website-away-wp-engine/

1. Use bv-pantheon plugin?
1. Create restore point
2. Download archive
3. Unpack
5. FTP download of `wp-content/uploads`
5. Be aware of filesize limitations
4. Add `wp-content/uploads` 
5. rm -rf the files listed in Brian Jackson's article
6. Compress archive(s)
7. Import
8. Verify success and resume `sites/migrate`

## Expected Audience
WPEngine users

## Path
`source/docs/articles/sites/migrate/from-wpengine`)

### PR DESCRIPTION
**Pull request by nataliejeremy**
Closes #847 

First draft of a new migrate doc. @bmackinney @ttrowell @rachelwhitton Please review and add more info to the doc as needed. Thanks!

### CODE REVIEW

File: source/docs/articles/sites/migrate/migrating-from-wpengine.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  - developing
+keywords: wordpress, pantheon, wpengine
+---
+
+Manually migrating a site from WP Engine to Pantheon can be tricky, as you cannot use WP Engine’s backup feature to transfer your site. This is because the backup file excludes many of your site's files and directories. See the full list of [exclusions](https://wpengine.com/support/restore/).
+
+The easiest way to migrate your site is to use the [Pantheon Migration Plugin](https://wordpress.org/plugins/bv-pantheon-migration/) created by BlogVault.
+
+## Migrate Using the Pantheon Migration Plugin
+
```
Comment (bmackinney) [inline source/docs/articles/sites/migrate/migrating-from-wpengine.md]: Users need to create the WordPress site on Pantheon first.

File: source/docs/articles/sites/migrate/migrating-from-wpengine.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+1. You can install the plugin by uploading `bv-pantheon-migration` through the WordPress Plugins menu or to the `/wp-content/plugins/` directory.
```
Comment (bmackinney) [inline source/docs/articles/sites/migrate/migrating-from-wpengine.md]: Does the plugin get installed to the WPEngine site or to the Pantheon site, or to both?

File: source/docs/articles/sites/migrate/migrating-from-wpengine.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        Import code, database, and files after creating the site using a combination of command-line tools (git, mysql-cli, and rsync) or with Git and the Site Dashboard's workflow tool. See [Migrate to Pantheon: Manual Site Import](/docs/articles/sites/migrate/manual-site-import) for detailed instructions.
+7. Test your site. When the site's code, database, and files are all in place, verify everything is working as expected. At the Site Dashboard, click **Visit Development Site** for verification.
+
+## Considerations
+
+### Folder Permission Errors
+If you receive any permission errors, verify your current PHP handler. You may need to switch it to SuPHP or DSO, depending on which one was running on the previous side.
+
+### WP-Config.php File
+WP Engine uses a modified `wp-config.php` file located in the root of your WP install, so you may need to generate a new one. You can do this with the [wp-config.php Generator](https://generatewp.com/wp-config/).
```
Comment (bmackinney) [inline source/docs/articles/sites/migrate/migrating-from-wpengine.md]: They will need to use our [`wp-config.php`](https://github.com/pantheon-systems/wordpress/blob/master/wp-config.php).

File: source/docs/articles/sites/migrate/migrating-from-wpengine.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+**SFTP Server Address**: Enter the host address found in your Pantheon Dashboard under Connection Information.  
+**SFTP Username**: Enter the SFTP username found in your Pantheon Dashboard under Connection Information.  
+**SFTP Password**: This is the password you use to log in to your Pantheon Dashboard.  
+3. Click **Migrate**. The plugin will automatically verify your SFTP credentials and notify you if any issues are found.
+4. After the migration is complete, click the button to see the results of your migration and visit your Pantheon site.
+
+## Manually Migrate from WP Engine
+
+1. Create and download an archive of your site. Archives can be stored in a single file or as three separate files.
+2. Unzip your site's codebase, files, and database.
```
Comment (bmackinney) [inline source/docs/articles/sites/migrate/migrating-from-wpengine.md]: I don't think the files come with. Please double check. The next step indicates they must be manually downloaded separately from the archive.

**Comment by bmackinney** [comment]
@rachelwhitton I think it works. 
:ship: :shipit: :rocket:

**Comment by bmackinney** [comment]
I believe so. You can put a note like - "You may need to preserve logic in `wp-config.php`"

**Comment by rachelwhitton** [comment]
+1 I think we're good to go! Thanks for drafting this new doc @nataliejeremy!

**Pull request merged**